material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -1956,6 +2118,10 @@ div.button-wrapper > a.connectMetaMask {
   box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
 }
 
+body[data-md-color-scheme='slate'] article.md-content__inner.md-typeset img {
+  border: 1px solid var(--feedback-border-color);
```

Comment:
would be nice if we had a default border color so we're not using "feedback" here, like we usually have one default border color that is the predominantly used one. Then we can use it for little things like this, for `hr` dividers, etc.